wandb/wandb
# docs(launch): Remove stack trace when failing to connect to docker

Fixes [WB-12702](https://wandb.atlassian.net/browse/WB-12702)

Description
-----------
When the Docker daemon isn't connected, the launch agent prints out an intimidating stack trace that makes it harder to find the actual issue. This change replaces the stack trace with an informative message.

![image](https://user-images.githubusercontent.com/39655500/223510225-6f26d3e7-3bd0-4286-b173-db15e659a78e.png)

Testing
-------
- Turn off docker
- Run launch agent, polling a queue with an item
- See above image for result


[WB-12702]: https://wandb.atlassian.net/browse/WB-12702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
